src/dotnet/Chat.UI.Blazor/Components/ChatList/ChatListTabUnreadCount.razor

```diff
@@ -23,14 +23,16 @@
 
     protected override ComputedState<Model>.Options GetStateOptions()
         => new() {
-            InitialValue = new (),
+            InitialValue = new(),
             UpdateDelayer = FixedDelayer.Get(0.1),
         };
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
-        var count = await ChatListUI.GetUnreadChatCount(ChatListView.PlaceId, ChatListFilter, cancellationToken).ConfigureAwait(false);
+        var placeId = ChatListView.PlaceId;
+        var chatListFilter = ChatListFilter;
+        var count = await ChatListUI.GetUnreadChatCount(placeId, chatListFilter, cancellationToken).ConfigureAwait(false);
 
-        return new Model {
+        return new() {
```

Comment:
space

Reply:
I think for target typed new expression we don't use space